### ISSUE TITLE
Storage question

### ISSUE THREAD

**Issue by unlucio** [hassio-addons/addon-plex#30]
How do you guys use this exactly?
I fail to understand what is the point of this plugin if there's no way to feed it my existing media library (which is an awful limitation form hassio) O.o.
Using a raspberry pi there's no way I can load my media on the SD card: 2TB+ SD cards simply don't exist. (and yes, a PI is enough to stream media, I've been happily using a PI 3 for 4 years without any problem)

Even if I skip the SD size problem in a thought experiment:
How do you transfer your media? using a `docker cp` to the container? What if the container gets deleted for any reason? You just do it over and over again? O.o

I'm not complaining (again, the limitation is from above), but I really cannot think of a possible scenario, so I'd really love to know one! :)

thanks for humoring my curiosity

**Comment by frenck** [comment]
When using the Home Assistant Operating System, mounting external media is not possible.

**Comment by unlucio** [comment]
So how is this plugin used? That was the whole point of my question.

**Comment by sinclairpaul** [comment]
When not using the Home Assistant Operating System, mounting external media is possible.

But in all honesty I would suggest using the forums/Discord for these kind of questions.

**Comment by unlucio** [comment]
From the search I ran (before aking the question), forums suggest to spin up a normal plex container from portainer (in order to pass your own volumes), as everything launched by the assio supervisor lacks the ability of mounting volumes even when providing "your own OS". That is what confused me further.
I come from a raspbian+hassio (dockerized) based config, and since I found it too cumbersome  I'm trying out the OS image in the hope things would be better integrated but I guess they're not as I expected.

What would be the best way to install hassio then? Should I just ditch the containerized environment (I'd rather not, I love docker) in order to ditch the limitations? 

PS: for discord I would happily avoid installing the 1000000nd chat app in my life :P
